x-pack/plugins/security_solution/public/management/components/artifact_list_page/artifact_list_page.tsx

```diff
@@ -68,6 +68,9 @@ export interface ArtifactListPageProps {
   searchableFields?: MaybeImmutable<string[]>;
   flyoutSize?: EuiFlyoutSize;
   'data-test-subj'?: string;
+  allowCardEditAction?: boolean;
+  allowCardDeleteAction?: boolean;
```

Comment:
Although delete is not currently needed, I just backed it in since we will likely need it in the future with RBAC and the ability to have View only pages under Management